conductor_api/src/conductor/admin.rs

```diff
@@ -209,12 +209,11 @@ impl ConductorAdmin for Conductor {
                     .into(),
             },
         };
-        new_config.instances.push(new_instance_config);
-        new_config.check_consistency(&mut self.dna_loader)?;
-        let instance = self.instantiate_from_config(id, Some(&mut new_config))?;
+        self.config.instances.push(new_instance_config);
+        self.config.check_consistency(&mut self.dna_loader)?;
```

Comment:
See my comment on the other PR about these lines, @maackle: https://github.com/holochain/holochain-rust/pull/1724#discussion_r330476516